### ISSUE TITLE
Questions about wandb

### ISSUE THREAD

**Issue by vicye07** [YRYangang/RRec#9]
really nice work and i am trying to run through the code. I just saw a closed issue where you provided the wandb log.May i know does the code include using wandb or we have to manually add to monitor the training process and the metric?

**Comment by YRYangang** [comment]
Hi there,

For huggingface trainers (including this repo), wandb implementation can be as simple as:

```python
os.environ["WANDB_PROJECT"] = "<my-amazing-project>"  # name your W&B project

from transformers import TrainingArguments, Trainer

args = TrainingArguments(..., report_to="wandb")  # turn on W&B logging
trainer = Trainer(..., args=args)
```

The corresponding code you need to change in this repo is:
https://github.com/YRYangang/RRec/blob/1f940c8ed4f2f4c9a21ba424be248314ea02a082/train.py#L193

See [official document](https://docs.wandb.ai/models/integrations/huggingface) for more details.

**Comment by vicye07** [comment]
Really appreciate!
